lib/hooks/useFacets/useFacets.js

```diff
@@ -133,8 +133,9 @@ const useFacets = ({
     setSearchParams({
       query: buildQuery(),
       facet: getFacetCql(name).all,
+      qindex,
```

Comment:
I don't think `qindex` is a required parameter for facets endpoint. We can probably don't add it to the request, but just add it to the key array in `useFacetsQuery` to force a new call when it changes

Reply:
`qindex` is not used for search parameters, it is only used for key array (queryKey).